packages/main/src/plugin/index.ts

```diff
@@ -609,7 +609,7 @@ export class PluginSystem {
     flowManager.init();
 
     const chatManager = new ChatManager(providerRegistry, mcpManager, this.getWebContentsSender, this.ipcHandle);
-    chatManager.init();
+    await chatManager.init();
```

Comment:
I have not catched that earlier, but ChatManager instance should be instantiated by Inversify